android/kotlin-central-login-oidc/app/src/main/java/com/example/app/Alert.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024. PingIdentity. All rights reserved.
```

Comment:
Sorry, I missed that one in my previous review, I think the correct is:

Copyright (c) 2024 Ping Identity. All rights reserved.

Reply:
done